.github/ISSUE_TEMPLATE/versioned_release.md

```diff
@@ -15,6 +15,7 @@ assignees: ""
 ```[tasklist]
 ## Release Checklist
 - [ ] Set a release date & notify team
+- [ ] Update our CITATION.cff file with the new release date and current Catalyst membership.
```

Comment:
```suggestion
- [ ] Update our CITATION.cff file with the new release date and current Catalyst membership.
- [ ] Update our .zenodo.json file with current Catalyst membership.
```

It would be nice to figure out how to have these two lists just be one list, but manual sync seems fine for now since it's very slow-moving.